.tool-versions

```diff
@@ -1 +1 @@
-elixir 1.2
+elixir 1.9.1
```

Comment:
Looking at this diff, I'd rather have the file deleted altogether. Can you `git rm` it pls and add it to `.gitignore`? 🙏

Reply:
I'm new to elixir, curious why we'd want this removed? It doesn't effect production installs and provides a helpful hint for future collaborators as to what version of elixir they should work with. Additionally, if folks have `asdf` installed they'll need this file in place, so it removes another step for any future folks to get started.

I'll make the change, but curious to learn the 'why' behind the best practices here!

Reply:
I hear you about making it easier to get future contributors started and that's a valid point. However, this file will not age well: Elixir is moving fast and it might not be the most up to date version anymore only a couple of weeks from now. To constantly keep updating this file creates diff noise. That being said, I don't know if it's "best practice", it's just my reasoning that it's more part of a developer's local setup.